Packages/io.chainsafe.web3-unity/Editor/ServerSettings.cs

```diff
@@ -196,6 +199,18 @@ private void OnGUI()
         chainID = EditorGUILayout.TextField("Chain ID: ", chainID);
         symbol = EditorGUILayout.TextField("Symbol: ", symbol);
         blockExplorerUrl = EditorGUILayout.TextField("Block Explorer: ", blockExplorerUrl);
+        enableAnalytics = EditorGUILayout.Toggle("Collect Data for Analytics: ", enableAnalytics);
```

Comment:
forgot the tooltip...replace text with `new GUIContent("Collect Data for Analytics:", "Consent to collecting data for analytics purposes. This will help improve our product.")`

Reply:
oh cool, hovering tooltip

Reply:
done